### PR TITLE
Volunteer to review and collaborate

### PR DESCRIPTION
**Pull request by Fedosin**
I'm putting my name out there to help review openstack cloud provider.

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -2,9 +2,11 @@ approvers:
 - anguslees
 - NickrenREN
 - dims
+- Fedosin
```
Comment (dims) [inline OWNERS]: Mike, let's add you as a reviewer first and then move to owner later. ok?

Comment (Fedosin) [reply OWNERS]: Sure! Done.

**Comment by hogepodge** [comment]
@Fedosin please resolve conflicts. Also, if we can alphabetize the list it could help with preventing merge conflicts and keep it tidy.

Thanks!

**Pull request closed without merging**